google-account-plugin/testSrc/com/google/gct/idea/AccountPluginInitializationComponentTest.java

```diff
@@ -0,0 +1,85 @@
+package com.google.gct.idea;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static org.mockito.Mockito.when;
+
+import com.google.common.collect.Lists;
+import com.google.gct.idea.util.IntelliJPlatform;
+import com.google.gct.idea.util.PlatformInfo;
+
+import com.intellij.mock.MockApplication;
+import com.intellij.openapi.Disposable;
+import com.intellij.openapi.application.Application;
+import com.intellij.openapi.application.ApplicationManager;
+import com.intellij.util.PlatformUtils;
+
+import java.util.List;
+import java.util.Properties;
+
+/**
+ * Tests to validate initialization on supported platforms
+ */
+public class AccountPluginInitializationComponentTest {
+
+  private Properties properties;
+
+  @Before
+  public void setUp() {
+    properties = System.getProperties();
+  }
+
+  @After
+  public void tearDown() {
+    System.setProperties(properties);
```

Comment:
I've been setting up the same properties in all tests, but I see your point, will change to beforeTest/afterTest